tests/test_load_emmo.py

```diff
@@ -6,7 +6,10 @@ def test_load_emmo() -> None:
     thisdir = os.path.abspath(os.path.dirname(__file__))
     sys.path.insert(1, os.path.abspath(os.path.join(thisdir, '..')))
     from ontopy import get_ontology  # noqa: E402, F401
+    from emmopy import emmo
 
+    onto = get_ontology()
+    onto.load()
 
-    emmo = get_ontology()
-    emmo.load()
+    emmo = emmo()
+    emmo_inferred = emmo(inferred=False)
```

Comment:
You're overwriting the variable for the function `emmo` with the returned value from said function here.